### PR TITLE
Update file structure for cody

### PR DESCRIPTION
**Pull request by abeatrix**
This PR removes all the unused and outdated directories from the `client/cody` project as they have been integrated into the main app, and moves the VS Code extension to the base `client/cody` directory.

I have also updated the README.md in `client/cody` to reflect the changes.

All CSS files are now using relative units instead of hard-coded units. Everything should scale according to user's system.

Some minor CSS fixes. 

<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
<tr>
<td>
image
</td>
<td>
image
</td>
</tr>
</table>

<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
<tr>
<td>
image
</td>
<td>
image
</td>
</tr>
</table>

Replaced placeholder logo with logo for cody:
image

## Test plan

Run the following commands from the **root** of this repository in this branch:
- pnpm --filter cody run build
- pnpm --filter cody run test:unit
- pnpm --filter cody run test:integration
- pnpm --filter cody run vsce:package

Everything should work as before the move.

## App preview:

- [Web](https://sg-web-bee-cody-restructure.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/cody/package.json [hunk 1]
```diff
@@ -1,14 +1,230 @@
 {
-  "name": "@sourcegraph/cody",
```
Comment (novoselrok) [inline client/cody/package.json:2]: We can probably rename this to `cody` now as well. And we can change the `displayName` and the `publisher`.

File: pnpm-workspace.yaml [hunk 2]
```diff
@@ -1,5 +1,4 @@
 packages:
-  - 'client/*'
```
Comment (novoselrok) [inline pnpm-workspace.yaml:2]: [Nit] Unnecessary change from single quotes to double quotes.

File: pnpm-lock.yaml
Comment (novoselrok) [inline pnpm-lock.yaml:1]: Why are there so many changes to the lock file? Are there any dependencies added or removed in this PR?

Comment (abeatrix) [reply pnpm-lock.yaml:1]: it was generated by pnpm i after removing the lock file. I've reset it to the one on cody-main and the pnpm generate command is passing again it seems

File: pnpm-workspace.yaml (hunk 2 above)
Comment (abeatrix) [reply pnpm-workspace.yaml:2]: that was done on save 😓
